platform/jewel/markdown/core/src/test/kotlin/org/jetbrains/jewel/markdown/styling/OrderedListNumberFormatStylesTest.kt

```diff
@@ -0,0 +1,95 @@
+package org.jetbrains.jewel.markdown.styling
+
+import org.jetbrains.jewel.markdown.rendering.MarkdownStyling
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertThrows
+import org.junit.Test
+
+public class OrderedListNumberFormatStylesTest {
+
+    @Test
+    public fun `numberFormatStyles should use firstLevel as default for others`() {
+        val styles =
+            MarkdownStyling.List.Ordered.NumberFormatStyles(
+                firstLevel = MarkdownStyling.List.Ordered.NumberFormatStyles.NumberFormatStyle.Decimal
+            )
+
+        assertEquals("1", styles.firstLevel.formatNumber(1))
+        assertEquals("2", styles.secondLevel.formatNumber(2))
+        assertEquals("3", styles.thirdLevel.formatNumber(3))
+    }
+
+    @Test
+    public fun `numberFormatStyles should use the respective format level as declared`() {
```

Comment:
This test is essentially checking that the property is set correctly, which in turn means you’re testing the compiler... the tests below already cover the meaningful part of this test (the `formatNumber` call result)